app/src/main/java/com/duckduckgo/app/browser/favicon/DuckDuckGoFaviconManager.kt

```diff
@@ -93,6 +95,20 @@ class DuckDuckGoFaviconManager constructor(
         }
     }
 
+    override suspend fun tryFetchFaviconForUrl(url: String): File? {
```

Comment:
could this be private?

Reply:
No, this is used to prefetch favicons from the browser and update the tab. It’s not a new addition